src/compile/legend/parse.ts

```diff
@@ -126,7 +126,6 @@ function getProperty(
       return (
         getFirstDefined(
           specifiedLegend.title, // legend title has higher precedence than fieldDef title
-          fieldDef.title,
```

Comment:
Why not simply using `fieldDefTitle(fieldDef, model.config) || undefined`?

(No need to use `getFirstDefined` anymore)

Reply:
I believe making that change would modify the function's behavior as it would no longer consider `specifiedLegend.title`. We could probably change it to `specifiedLegend.title || fieldDefTitle(fieldDef.title, model.config)`

Reply:
specifiedLegend is fieldDef.legend

So what I suggested is right